app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -120,9 +120,8 @@ def check_enrollment(enrollment)
     enrollment.update(status_check_attempted_at: status_check_attempted_at)
   end
 
-  def passed_with_unsupported_secondary_id_type?(enrollment, response)
-    return enrollment.capture_secondary_id_enabled &&
-           response['secondaryIdType'].present? &&
+  def passed_with_unsupported_secondary_id_type?(_enrollment, response)
```

Comment:
if `enrollment` is no longer used, can we refactor this out of the method signature? (and the callsite too)

```suggestion
  def passed_with_unsupported_secondary_id_type?(response)
```

Reply:
ahh yes, thank you for catching this! i'll update